### DISCUSSION TITLE
newSessionBegan vs. TelemetryDeck.Session.started

### DISCUSSION THREAD

**Discussion by joebez** [TelemetryDeck/Viewer#442] [Q&A] [answered]
When launched, my (in development) iOS app seems to be sending both the `newSessionBegan` and `TelemetryDeck.Session.started` signals. I am using the latest version of the TelemetryClient (2.3.0) now, but the app originally used an earlier version of TelemetryDeck.

Should both of these signals be sent? It seems like the more modern `TelemetryDeck.Session.started` would be sufficient, and I'm concerned my account is registering two times the number of signals on every launch.

If both of them shouldn't be sent, is there a way to disable the sending of the older `newSessionBegan` signal?

**Comment by joebez** [comment]
Closing as answered. From the "SwiftSDK 2.0" [blog post](https://telemetrydeck.com/blog/grand-rename/):

> To make sure these changes don’t affect existing customers with existing insights anytime soon, we decided to keep sending the old names for a transition period of at least 1 year while also sending the new ones.

**Comment by winsmith** [comment, marked as answer]
Originally we thought it would be a good idea to continue sending both signals for backwards compatibility. However, we heard from customers that it's annoying that their signal count is going up a lot because of this change. We've therefore made the decision to no longer include the old "newSessionBegan" signal starting in SwiftSDK 2.4.0
